### PR TITLE
Add Libmypaint/Brushlib

### PR DESCRIPTION
**Pull request by superloach**
Working toward #2580.
We're almost done, this is exciting!

### CODE REVIEW

File: packages/libmypaint-gegl.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Libmypaint < Package
+  description 'This is the brush library used by MyPaint. A number of other painting programs use it too.'
```
Comment (uberhacker) [inline packages/libmypaint-gegl.rb]: Better description?  `Libmypaint is MyPaint’s brushstroke rendering code, in a form that can be used by other programs.`

Comment (superloach) [reply packages/libmypaint-gegl.rb]: Good idea, I just used what I found on the GitHub.

File: packages/libmypaint-gegl.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Libmypaint < Package
```
Comment (uberhacker) [inline packages/libmypaint-gegl.rb]: The class name should match the filename.  Rename `libmypaint-gegl.rb` to `libmypaint.rb`?

Comment (superloach) [reply packages/libmypaint-gegl.rb]: I had originally called it `libmypaint.rb`, but remembered it was actually called `libmypaint-gegl`.
Should I revert to `libmypaint.rb`, or change the class to `Libmypaint_gegl`?

Comment (uberhacker) [reply packages/libmypaint-gegl.rb]: Either way is fine as long as the class matches the filename, otherwise, crew will get confused.

Comment (superloach) [reply packages/libmypaint-gegl.rb]: I'll just omit the `gegl`, keep it shorter.

**Comment by superloach** [comment]
I'm just eager to get this working 😁

**Pull request merged**